{{cookiecutter.github_repository}}/tests/integration/test_auth.py

```diff
@@ -40,3 +42,62 @@ def test_user_login(client):
         'id', 'email', 'name', 'auth_token'
     ]
     assert set(expected_keys).issubset(response.data.keys())
+
+
+def test_user_password_change(client):
+    url = reverse('auth-password-change')
+    current_password = 'password1'
+    new_password = 'paSswOrd2.#$'
+    user = f.create_user(email='test@example.com')
+    user.set_password(current_password)
+    user.save()
```

Comment:
You can pass in `password` to `create_user` directly see factories.py for details.

Reply:
Nice. I am changing this for other tests too.